builtin/logical/pkiext/pkiext_binary/pki_cluster.go

```diff
@@ -67,6 +71,39 @@ func (vpc *VaultPkiCluster) GetActiveNode() *api.Client {
 	return vpc.cluster.Nodes()[0].APIClient()
 }
 
+func (vpc *VaultPkiCluster) AddNameToHostsFile(ip, hostname string, logConsumer func(string), logStdout, logStderr io.Writer) error {
+	updateHostsCmd := []string{
+		"sh", "-c",
+		"echo '" + ip + " " + hostname + "' >> /etc/hosts",
```

Comment:
I think I'd like to avoid this and go with the new containerized DNS server in the future, but we'll get there :-)